.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
I think let's allow blank issues just on the off-chance that we don't have something covered, and we can just close ones that are done and redirect them to the form.